docker-multinode/common.sh

```diff
@@ -123,40 +125,12 @@ kube::multinode::detect_lsb() {
   kube::log::status "Detected OS: ${lsb_dist}"
 }
 
-# Start a docker bootstrap for running etcd and flannel
-kube::multinode::bootstrap_daemon() {
```

Comment:
hmmm..the code for post-1.4  and pre-1.4 versions would not share much in common - not so nice....I would suggest to drop support for old Kubernetes versions...